### DISCUSSION TITLE
не сетятся данные

### DISCUSSION THREAD

**Discussion by IrinaG1090** [totumonline/totum-ru-issues-and-discussions#462] [Как это сделать?] [answered]
в  первой расчетной таблице  с поле1, поле2, поле3 есть кнопка с кодом действия 

a1= : insert(table: 'rem'; inserts: "inserts"; log: true)

a2=: set(table: 'remonttt'; cycle: $#inserts[0]; field: 'pole1' = #pole1; field: 'pole2' = #pole4; field: 'pole3' = #pole5; log: true)

цикл создается, но вот set не работает. Не сетятся данные в новую расчетную таблицу из первой таблицы

Как можно еще этот момент решить или как и что поправить - подсветите пожалуйста

**Comment by k1erk** [comment]
cycle: $#inserts[0] обращение к ключу не нужно, там простое число.

**Reply by ttm-alexeypolunin** [reply]
$#inserts[0] верное обращение тк в inserts — list

**Comment by ttm-alexeypolunin** [comment, marked as answer]
`pole1,2,3` в этом коде в `remonttt`  должны быть в хедере. Если они в строчной части, то set неприменим.

Ну и смотрите отладку
